File: FizzBuzz/FizzBuzz.php
<?php
// FizzBuzz Kata 
/*   
if i is a multiple of 3 print "fizz"
else if i is a multiple of 5 print "buzz"
else if i is a multiple of 3 and a multiple of 5 print "fizzbuzz"
else print i
*/

// Wrapped in a function for JSLitmus purposes

function fizzbuzz($maxNumber) {
//Iterate through integers and apply calculations
    for ($count = 1; $count <= $maxNumber; $count++) {
        if ($count % 15 === 0) {
            echo "fizzbuzz <br/>";
        }
        elseif ($count % 3 === 0) {
            echo "fizz <br/>";
        }
        elseif ($count % 5 === 0) {
            echo "buzz <br/>";
        }
        else {
            echo $count."<br/>";
        }
    }
}
?>

File: FizzBuzz/fizzbuzz.cs
//fizzbuzz Kata  
//if i is a multiple of 3 print "fizz"
//else if i is a multiple of 5 print "buzz"
//else if i is a multiple of 3 and a multiple of 5 print "fizzbuzz"
//else print i

//C# Implementation


using System;

namespace Kata
{
    class fizzbuzz
    {
        static void Main(string[] args)
        {
        for (int count = 1; count <= 100; count++)
        {
            if (count % 5 == 0 && count % 3 == 0)
            {
                Console.WriteLine("fizzbuzz");
            }
            else if (count % 5 == 0)
            {
                Console.WriteLine("buzz");
            }
            else if (count % 3 == 0)
            {
                Console.WriteLine("fizz");
            }
            Console.WriteLine(count);
        }
        Console.ReadLine();
	    }
    }
}